### ISSUE TITLE
Automated PHEC GISAID submissions

### ISSUE THREAD

**Issue by SamStudio8** [COG-UK/dipi-group#53]
CLIMB-COVID will be handling PHEC submissions to GISAID and ENA going forward, but we first need to resolve previous submissions to ensure that our catalogue of accessions is complete; and that we do not inadvertently send any duplicate genomes to GISAID with the COG IDs rather than the PHE ones.

AL at PHE has sent me the requisite mapping.

**Comment by SamStudio8** [comment]
I've reconciled the existing accession list from PHEC with Majora and sent back a few queries to untangle.

**Comment by SamStudio8** [comment]
AL has returned the table of sequence queries for me to go through

**Comment by SamStudio8** [comment]
1515 new accessions synchronised to Majora as of 2021-05-27. There are a few accessions left to untangle but I don't think these should block the more important task of getting the automated submissions turned on. I'll do a dry-run for GISAID and see which duplicates Majora tries to submit.

**Comment by SamStudio8** [comment]
A dry run shows there are approximately 16,000* PHEC genomes to be added to GISAID. I was a little surprised by this number, but after doing some double checking this does seem to be right. I've run the IDs for the samples to be uploaded against my list of GISAID accessions and there are no overlaps (although this is only partly convincing).

It is likely we'll inadvertently add a few duplicates but I've done my best to mitigate the worst of it, and it won't be on a significant order of magnitude like had we not intervened to synchronise the accessions first.

I have instructed PHE they are able to re-enable their opt-in, so I'll close this issue.

Update: Spoke too soon, seems the backlog is indeed unexpected according to AL. I've shared the table of staged sequences to try and get to the bottom of it, but leaving this issue closed unless we need to take action on our end.
